src/tools/sfdk/sdkmanager.cpp

```diff
@@ -1318,7 +1318,7 @@ QString SdkManager::cleanSharedTarget() const
     QString errorString;
     const BuildTargetData target = SdkManager::configuredTarget(&errorString);
     if (!target.isValid()) {
-        qerr() << errorString << endl;
+        qCDebug(sfdk) << errorString << endl;
```

Comment:
I would prefer to let `cleanSharedTarget` accept and forward `errorString` argument, then print the debug message only where it is used, i.e. in `mapEnginePath`.

Now when `qCDebug` is used, it would be better to add some contextual information too, so maybe

```
qCDebug(sfdk).noquote() << "Not mapping shared target path:" << errorString
```

Reply:
Fixed